engine/src/main/java/org/hibernate/validator/internal/util/logging/Log.java

```diff
@@ -228,10 +246,10 @@ UnexpectedTypeException getNoValidatorFoundForTypeException(String constraintTyp
 	@Message(id = 63, value = "Unable to instantiate %s.")
 	ValidationException getUnableToInstantiateException(String className, @Cause Exception e);
 
-	ValidationException getUnableToInstantiateException(Class<?> clazz, @Cause Exception e);
```

Comment:
Yup, I was wondering if there was some magic to get the message from the previous one (as I would expect them to be the same)?